app/ui/src/app/customizations/api-connector/api-connector-detail.component.ts

```diff
@@ -1,30 +1,33 @@
 import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
-import { Observable } from 'rxjs/Observable';
+import { ActivatedRoute } from '@angular/router';
 
-import { ApiConnector, ApiConnectorData } from './api-connector.models';
+import { ApiConnectorData } from './api-connector.models';
 import { ApiConnectorStore } from './api-connector.store';
+import { ApiConnectorService } from '@syndesis/ui/customizations/api-connector/api-connector.service';
```

Comment:
Referencing paths like this does not benefit future refactorings. The point for using aliased barrels is precisely not to define paths to the end file. Here you should:

1. Reference the file by its relative path (ultimately both files belong in the same domain): `'./api-connector.service'`
2. Add `api-connector.service` to the index barrel and reference it like `'@syndesis/ui/customizations/api-connector'`. This will work fine **as long as** you don't have to expose (this is, export) this same file outside the `customizations/api-connector`. Otherwise you might wind up creating an undesired circular reference.